bundle/Resources/translations/ezrepoforms_policies.en.xlf

```diff
@@ -0,0 +1,86 @@
+    <header>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="3742f2cc514848bd4a5eb03430e412d44f5d9ec2" resname="policy.limitation.identifier.class">
+        <source>Content Type</source>
+        <target>Content Type</target>
+        <note>key: policy.limitation.identifier.class</note>
+      </trans-unit>
+      <trans-unit id="76adf2a27f1ae0ab14b623729cd3f281a6e2c285" resname="policy.limitation.identifier.group">
+        <source>Group</source>
+        <target>Group</target>
+        <note>key: policy.limitation.identifier.group</note>
+      </trans-unit>
+      <trans-unit id="cee78b25ec2de77cf09109e6a2ac175a515a8350" resname="policy.limitation.identifier.language">
+        <source>Language</source>
+        <target>Language</target>
+        <note>key: policy.limitation.identifier.language</note>
+      </trans-unit>
+      <trans-unit id="ab9163a435d32610a7a1af8df07ab38eebc4c7c6" resname="policy.limitation.identifier.newsection">
+        <source>New Section</source>
+        <target>New Section</target>
+        <note>key: policy.limitation.identifier.newsection</note>
+      </trans-unit>
+      <trans-unit id="a9958f1d76f4685bf5fc32eee7514046db942160" resname="policy.limitation.identifier.newstate">
+        <source>New State</source>
+        <target>New State</target>
+        <note>key: policy.limitation.identifier.newstate</note>
+      </trans-unit>
+      <trans-unit id="be2b0449251f53db814dc4fbbb27e534419add2b" resname="policy.limitation.identifier.node">
+        <source>Location</source>
+        <target>Location</target>
+        <note>key: policy.limitation.identifier.node</note>
+      </trans-unit>
+      <trans-unit id="9daf8e1a94a6409f9b9b102e042c2858e4307ea8" resname="policy.limitation.identifier.owner">
+        <source>Owner</source>
+        <target>Owner</target>
+        <note>key: policy.limitation.identifier.owner</note>
+      </trans-unit>
+      <trans-unit id="e3773b71c8e7fda97f221ab25a6c25164d179d84" resname="policy.limitation.identifier.parentclass">
+        <source>Parent Content Type</source>
+        <target>Parent Content Type</target>
+        <note>key: policy.limitation.identifier.parentclass</note>
+      </trans-unit>
+      <trans-unit id="39bf0d9c1e0e8b64193bf09d83fbd89c2dac3421" resname="policy.limitation.identifier.parentdepth">
+        <source>Parent Depth</source>
+        <target>Parent Depth</target>
+        <note>key: policy.limitation.identifier.parentdepth</note>
+      </trans-unit>
+      <trans-unit id="cb52ff8582459d1bc981e9f98cf1c79876f83a1e" resname="policy.limitation.identifier.parentgroup">
+        <source>Parent Group</source>
```

Comment:
Same here, this is effectively "Content Type Group of Parent Location", however thats a bit long to list in the short version perhaps.

Reply:
`Content Type Group of Parent` is shorter